api/app/analysis/dist_alerts/analysis.py

```diff
@@ -136,13 +152,18 @@ async def zonal_statistics(aoi, geojson, intersection=None):
         alerts_df.ldacs_driver = alerts_df.ldacs_driver.apply(
             lambda x: DIST_DRIVERS.get(x, "Unclassified")
         )
+    elif intersection == "grasslands":
+        alerts_df["grasslands"] = alerts_df["grasslands"].apply(
+            (lambda x: "grasslands" if x == 1 else "non-grasslands"), meta=("grasslands", "uint8")
+        )
+        alerts_df = alerts_df.drop("year", axis=1).reset_index(drop=True)
 
     alerts_df = alerts_df[alerts_df.value > 0]
     return alerts_df
 
 
-async def get_precomputed_statistics(aoi, intersection, dask_client):
-    if aoi["type"] != "admin" or intersection not in [None, "natural_lands", "driver"]:
+async def get_precomputed_statistics(aoi, intersection: Optional[str], dask_client):
+    if aoi["type"] != "admin" or intersection not in [None, "natural_lands", "driver", "grasslands"]:
```

Comment:
Minor so you can ignore but I think we can rely on the intersection spec in the DistAnalyticsIn model to handle invalid intersections at the endpoint level or we can use one list in both places.